lib/events/dynamoevents/dynamoevents.go

```diff
@@ -93,6 +109,12 @@ func (cfg *Config) CheckAndSetDefaults() error {
 	if cfg.UIDGenerator == nil {
 		cfg.UIDGenerator = utils.NewRealUID()
 	}
+
+	// Table is required.
```

Comment:
Not sure why you'd move the test after the defaults - usually defaults are applied to a valid (empty) state, so it looked to have been fine before?